ldaptor/protocols/pureldap.py

```diff
@@ -212,11 +212,11 @@ def toWire(self):
             ], tag=self.tag).toWire()
 
     def __repr__(self):
-        converter = to_bytes if six.PY2 else to_unicode
+        auth = '*' * len(self.auth)
         l = []
         l.append('version=%d' % self.version)
-        l.append('dn=%s' % repr(converter(self.dn)))
-        l.append('auth=%s' % repr(converter(self.auth)))
+        l.append('dn=%s' % repr(repr_converter(self.dn)))
```

Comment:
Not in all cases unfortunately. In Python 3 it may look as `dn="b'uid=user'"` if `dn` was specified as byte string. `repr_converter` is used to make it look like `dn="uid=user"`.
`repr` function is also needed, without it the result may look like `dn=uid=user`.

Reply:
omg :) ok. thanks for the comment
this py3/py2 thing is ugly.

I think that this can be merge as it is and have a release